frontend/demo/component/avatar/avatar-sizes.ts

```diff
@@ -0,0 +1,24 @@
+import '../../init'; // hidden-full-source-line
+
+import { html, LitElement, customElement } from 'lit-element';
+import '@vaadin/vaadin-avatar/vaadin-avatar';
+import { applyTheme } from 'themes/theme-generated.js';
+@customElement('avatar-basic')
```

Comment:
Conflicting custom element name
```suggestion
@customElement('avatar-sizes')
```